internal/opentf/transform_targets.go

```diff
@@ -42,13 +44,27 @@ func (t *TargetsTransformer) Transform(g *Graph) error {
 		}
 	}
 
+	if len(t.Excludes) > 0 {
+		targetedNodes, err := t.selectTargetedNodes(g, t.Excludes, true)
```

Comment:
a variable name of `excludedNodes` would preserve cognitive context when reading through the code.

Throughout this change we have two categories:  targeted and excluded.  Here we seem to cross the streams, potentially creating confusion.

L54-57 becomes:
```
			if excludedNodes.Include(v) {
				log.Printf("[DEBUG] Removing %q, filtered by excludes.", dag.VertexName(v))
				g.Remove(v)
```
and later readers of the code don't have to ask themselves why targeted nodes are being removed.